third_party/java/apkbuilder/BUILD.tools

```diff
@@ -1,15 +0,0 @@
-# The build file for apkbuilder when it's in bazel's embedded tools repo.
-
-licenses(["notice"])  # Apache 2.0
-
-java_binary(
```

Comment:
Did you check whether that file is referenced from any other place?

Reply:
I couldn't find any reference to [third_party/java/apkbuilder](https://github.com/bazelbuild/bazel/search?q=third_party%2Fjava%2Fapkbuilder&unscoped_q=third_party%2Fjava%2Fapkbuilder) except in third_party/BUILD